pkgs/development/ocaml-modules/bls12-381/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, fetchFromGitLab, buildDunePackage, ff, dune-configurator, zarith, ctypes, tezos-rust-libs, ocaml, alcotest }:
+
+buildDunePackage rec {
+  pname = "bls12-381";
+  version = "0.3.15";
+
+  src = fetchFromGitLab {
+    owner = "dannywillems";
+    repo = "ocaml-bls12-381";
+    rev = version;
+    sha256 = "1s8n657fsl2gs01p7v2ffpcfzymavifhhpriyx1gq5qh4zvvw4vr";
+  };
+  useDune2 = true;
+
+  propagatedBuildInputs = [
+    ff
+    dune-configurator
```

Comment:
Is `dune-configurator` needed? It should not be propagated.

Reply:
Seems like it's not needed actually, removing it